From 76b3150199e4a6a52a360f99e2c757a99d10bee8 Mon Sep 17 00:00:00 2001
From: Hisashi Nakamura <hisashi.nakamura.ak@renesas.com>
Date: Fri, 26 Jul 2013 09:16:27 +0900
Subject: [PATCH 0748/1083] ARM: shmobile: lager: Increase kernel memory size
 in DTS

Lager kernel memory size is changed from 768Mbyte to 1792Mbyte.

Signed-off-by: Hisashi Nakamura <hisashi.nakamura.ak@renesas.com>
---
 arch/arm/boot/dts/r8a7790-lager.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7790-lager.dts b/arch/arm/boot/dts/r8a7790-lager.dts
index 646ef5a..788ea23 100644
--- a/arch/arm/boot/dts/r8a7790-lager.dts
+++ b/arch/arm/boot/dts/r8a7790-lager.dts
@@ -17,12 +17,12 @@
 	compatible = "renesas,lager", "renesas,r8a7790";
 
 	chosen {
-		bootargs = "console=ttySC6,38400 mem=768M ip=dhcp root=/dev/nfs ignore_loglevel";
+		bootargs = "console=ttySC6,38400 mem=1792M ip=dhcp root=/dev/nfs ignore_loglevel";
 	};
 
 	memory@40000000 {
 		device_type = "memory";
-		reg = <0 0x40000000 0 0x80000000>;
+		reg = <0 0x40000000 0 0xC0000000>;
 	};
 
 	lbsc {
-- 
1.8.3.2

